packages/terra-consumer-icon/src/IconBase.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
after pulling the IconBase from terra-icon we have one hiccup to pass the IconBase raw content to [PropsTable src ](https://github.com/cerner/terra-core/blob/master/packages/terra-props-table/src/PropsTable.jsx) to display props description in [example page](https://github.com/cerner/terra-consumer/pull/7/files#diff-48d8ead84efbb821ce305bebc52e8253R10). I will take this up with terra team. Currently we cannot pass the raw content of any target component from lib folder to PropsTable as src. cc @bjankord @mjhenkes